python/sqlflow_submitter/xgboost/predict.py

```diff
@@ -36,7 +36,8 @@ def pred(datasource,
         conn = db.connect_with_data_source(datasource)
     label_name = label_meta["feature_name"]
     dpred = xgb_dataset(datasource, 'predict.txt', select, feature_metas,
-                        feature_column_names, None, is_pai, pai_table, True)
+                        feature_column_names, None, is_pai, pai_table, True,
+                        True)  # NOTE: default to use external memory
```

Comment:
I'm not sure the reason for setting `True` as the default value.

Reply:
Because we cannot specify the attributed in the predict statement, using `True` as the default is to make the predict statement being capable of processing large datasets automatically.